docs/src/Text.doc.js

```diff
@@ -185,29 +185,23 @@ card(
     defaultCode={`
 <Box>
   <Box marginBottom={2}>
-    <Text inline size="sm">
-      {'Small'}
-    </Text>{' '}
+    <Text inline size="sm">Small </Text>
```

Comment:
@AlbertCarreras let's leave the `{' '}` after `</Text>` to make the space obvious

```
<Text inline size="sm">Small</Text>{' '}
```

Reply:
@AlbertCarreras fyi updated these myself